vmdb/app/models/ems_refresh/save_inventory_infra.rb

```diff
@@ -243,6 +243,7 @@ def save_ems_clusters_inventory(ems, hashes, target = nil)
     self.save_inventory_multi(:ems_clusters, EmsCluster, ems, hashes, deletes, :uid_ems, nil, :ems_children)
     self.store_ids_for_new_records(ems.ems_clusters, hashes, :uid_ems)
   end
+  alias_method :save_clusters_inventory, :save_ems_clusters_inventory
```

Comment:
Should I just change the name of the method?
Not sure how much legacy compatibility we want here.